decidim-accountability/app/commands/decidim/accountability/admin/update_imported_result.rb

```diff
@@ -5,84 +5,30 @@ module Accountability
     module Admin
       # This command is executed when the user changes a Result from the admin
       # panel.
```

Comment:
```suggestion
      # panel using the "Import results from CSV file" feature
```

Reply:
Done in 94771d78819c3002354367f20f75b28d814444b9